README.rst

```diff
@@ -1888,18 +1875,44 @@ for configuring and running SQLAlchemy in a production system.
     {
         Animal {
             name @output(out_name: "animal_name")
-                 @filter(op_name: "in_collection", value: ["$names"])
+            out_Animal_LivesIn {
+                name @output(out_name: "location_name")
+            }
         }
     }
     '''
-    parameters = {
-        'names': ['animal name 1', 'animal name 2'],
-    }
+    parameters = {}
 
     compilation_result = graphql_to_sql(sql_schema_info, graphql_query, parameters)
+
+    # WARNING: Strings compiled from SQLAlchemy ClauseElement objects may not be executable.
+    #          See https://stackoverflow.com/questions/5631078/sqlalchemy-print-the-actual-query
+    print(compilation_result.query.compile())
+    # SELECT "Animal_1".NAME   AS animal_name,
+    #       "Location_1".NAME AS location_name
+    # FROM   "animal" AS "Animal_1"
+    #       JOIN "location" AS "Location_1"
+    #         ON "Animal_1".location = "Location_1".uuid
+
     query_results = [dict(result_proxy) for result_proxy in engine.execute(compilation_result.query)]
 
 
+
+Querying Across SQL Databases
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For certain SQL database management systems, (namely Microsoft SQL Server), it is possible to
```

Comment:
I haven't seen a SQL dialect that doesn't have the concept of multiple schemas. You make it sound like this is a `SQL Server` problem.